src/redis-cli.c

```diff
@@ -5866,7 +5866,7 @@ static int clusterManagerCommandAddNode(int argc, char **argv) {
     added = 1;
 
     if (!master_node) {
-        // Send functions to the new node, if new node is a replica it will get the functions from his primary.
+        // Send functions to the new node, if new node is a replica it will get the functions from its primary.
```

Comment:
```suggestion
        /* Send functions to the new node, if new node is a replica it will get the functions from its primary. */
```

Reply:
@oranagra I saw that on redis-cli line comments are used all around, should I still change it?

Reply:
i suppose it has some line comments by mistake (too many different hands worked on it, and some maybe without aggressive review)
```
oran@Oran-laptop:~/work/redis$ grep -c "/\*" src/redis-cli.c
475
oran@Oran-laptop:~/work/redis$ grep -c "//" src/redis-cli.c
29
```

Reply:
Fixed